test/nstemplatetier/nstemplatetier.go

```diff
@@ -99,34 +99,40 @@ var CurrentBase1nsTemplates = toolchainv1alpha1.NSTemplateTierSpec{
 	},
 }
 
-// AppStudioTemplates current templates for the "appstudio" tier
-var AppStudioTemplates = toolchainv1alpha1.NSTemplateTierSpec{
+// AppStudioEnvTemplates current templates for the "appstudio-env" tier
+var AppStudioEnvTemplates = toolchainv1alpha1.NSTemplateTierSpec{
 	Namespaces: []toolchainv1alpha1.NSTemplateTierNamespace{
 		{
-			TemplateRef: "appstudio-dev-123456new",
+			TemplateRef: "appstudio-env-env-88b275d-88b275d",
```

Comment:
Is it supposed to be
```suggestion
			TemplateRef: "appstudio-env-88b275d-88b275d",
```
?

I guess it doesn't matter since the tests are passing but just caught my eye

Reply:
I just copied those from a running instance , but it doesn't actually matter.